### PR TITLE
added two extensions

### PR DESCRIPTION
**Pull request by scollovati**
Hi, 
I have added:
- KevinRose.vsc-python-indent
- njpwerner.autodocstring 

Both extensions have MIT License.

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -971,6 +976,11 @@
       "id": "Nightrains.robloxlsp",
       "repository": "https://github.com/NightrainsRbx/RobloxLsp"
     },
+    {
+      "id": "njpwerner.autodocstring",
+      "respository": "https://github.com/NilsJPWerner/autoDocstring",
+      "version": "0.5.4"
```
Comment (jankeromnes) [inline extensions.json:982]: No need to pin a specific version if we publish straight from the default branch.

```suggestion
      "respository": "https://github.com/NilsJPWerner/autoDocstring"
```

**Review by jankeromnes — APPROVED** [review]
Great additions! Thanks.

One small thing to adjust, but I can apply my own suggestion directly on GitHub before merging, so no extra work needed.

**Comment by scollovati** [comment]
thanks @jankeromnes. Actually I found a typo `"respository"` instead of `"repository"` :( My fault!

**Comment by jankeromnes** [comment]
Aha, oops, I missed that one.

FYI, feel free to use the script `node add-extension https://github.com/NilsJPWerner/autoDocstring` to add extensions in the future, as it will do most of the work automatically.

**Comment by jankeromnes** [comment]
Auto-publishing both extensions worked fine! 🎉
- https://open-vsx.org/extension/KevinRose/vsc-python-indent
- https://open-vsx.org/extension/njpwerner/autodocstring

@scollovati Please kindly confirm that both published extensions work as intended.

**Comment by scollovati** [comment]
@jankeromnes tested a few minutes ago. They are working perfectly!

**Pull request merged**